### PR TITLE
CI scripts

### PR DESCRIPTION
**Pull request by schu**

### CODE REVIEW

**Comment by iaguis** [comment]
Can you link to the tests?

**Comment by schu** [comment]
https://semaphoreci.com/schu/gobpf/branches/schu-ci
https://travis-ci.org/schu/gobpf

**Comment by schu** [comment]
Tests pass, good to go I think.

File: semaphore.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
```
Comment (alban) [inline semaphore.sh]: Can you add a comment to explain similar to 
https://github.com/weaveworks/tcptracer-bpf/blob/master/semaphore.sh

Comment (schu) [inline semaphore.sh]: Done.

**Comment by alban** [comment]
lgtm

**Pull request merged**